### ISSUE TITLE
[V3] Feat: make front-end adapts to `route` API interface modifications

### ISSUE THREAD

**Issue by SkyeYoung** [apache/apisix-dashboard#2636]
# Feature request

## Please describe your feature

The back-end API interface of APISIX Dashboard V3 has changed a lot compared to V2, so I think the front-end of Dashboard V3 should be adapted to the API interface changes as soon as possible.

First of all, we can start by adapting the API interface of the route part.

## Describe the solution you'd like

1. All updates should be developed under the `next` branch to prevent damage to the `master` branch.
2. First you need to get the V3 backend up and running to facilitate testing of the APIs.
3. Then you need to understand the API definition of V3 and the difference with V2 version.
	1. Since the Dashboard project has not been well documented in the past, you will need to look at the code yourself, or make a request to the backend, to understand the differences between these APIs.
	2. You can import the files in the following link into postman to see the definition of the new `route` APIs. (Thx @bzp2010 ) [postman_collection.zip](https://github.com/apache/apisix-dashboard/files/9765201/Basic.resources.CRUD.postman_collection.zip)
4. Next you can start adapting the new APIs, Don't forget tests.
5. During this time, you can create a Pull Request for community members to review and provide you with assistance as needed.
6. Finally, wait for the PR to merge and complete your first contribution to the community (or the Nth. XD)

## Describe alternatives you've considered

nothing.

## Additional context

nothing.

cc @bzp2010  @guoqqqi

**Comment by tomtiao** [comment]
I would like to implement this.

I found that many components are making request to v2 version of the apis, thus blocking the Cypress test of the `route` part. Is there anything I can do about it?

**Comment by SkyeYoung** [comment]
> I found that many components are making request to v2 version of the apis, thus blocking the Cypress test of the `route` part. Is there anything I can do about it?

Yes, this is a problem ...... I think if you can, when you deal with E2E tests, make a list of these codes that prevent the tests from passing. Then you can have community members help you evaluate the difficulty of these test changes to decide if you want to do all of them in one PR.

> I would like to implement this.

cc @guoqi @bzp2010

**Comment by juzhiyuan** [comment]
Refer to https://github.com/apache/apisix-dashboard/issues/2981
